crypto/keys/keybase.go

```diff
@@ -469,3 +465,12 @@ func addrKey(address types.AccAddress) []byte {
 func infoKey(name string) []byte {
 	return []byte(fmt.Sprintf("%s.%s", name, infoSuffix))
 }
+
+func (kb dbKeybase) deleteOfflineLedgerKey(info Info, yesPassphrase string) error {
+	if yesPassphrase != "yes" {
```

Comment:
This isn't too useful if we just hardcode "yes" in the caller (but I don't think we should, see above comment).

Reply:
Addressed